client/js/components/statement/assessmentTable/AssignEntityModal.vue

```diff
@@ -8,46 +8,44 @@
 </license>
 
 <template>
-  <portal to="vueModals">
-    <dp-modal
-      ref="assignModal"
-      @modal:toggled="handleClose"
-      content-classes="u-1-of-2">
-      <!-- modal header -->
-      <template v-slot:header>
-        {{ Translator.trans('assignment.entity.assign.to.other', { entity: Translator.trans(entityType) }) }}
-      </template>
-
-      <!--the height of the div below (220px) is needed because the multiselect dropdown will cause the modal to have a scroll when opened (see: https://github.com/shentao/vue-multiselect/issues/723). Once this github issue is solved and dropdown will overlay modal content, the height class and button top margin can be removed.
-      ATTENTION! To be able to reduce the dropdown's size, I had to change styling in _multiselect.scss (overflow and max-height props in dropdown__content). It may cause the comeback of T11129 bug -->
-
-      <!-- modal content -->
-      <div class="height-220">
-        <h3>{{ Translator.trans('user.choose') }}:</h3>
-        <div>
-          <dp-multiselect
-            :id="`r_${entityId}`"
-            v-model="selected"
-            :allow-empty="false"
-            class="u-n-ml-0_25"
-            :custom-label="option => `${option.name} ${option.id === currentUserId ? '(Sie)' : ''}`"
-            :name="`r_${entityId}`"
-            :options="[{ id: '', name: '-'}, ...users]"
-            :max-height="150"
-            track-by="id">
-            <template v-slot:option="{ props }">
-              {{ props.option.name }} {{ props.option.id === currentUserId ? ` (Sie)` : '' }}
-            </template>
-          </dp-multiselect>
-        </div>
-        <dp-button
-          class="u-mt float-right"
-          :busy="loading"
-          :text="Translator.trans('assignment.generic.assign.to.chosen', { entity: Translator.trans(entityType) })"
-          @click="assignEntity" />
+  <dp-modal
+    ref="assignModal"
+    @modal:toggled="handleClose"
+    content-classes="u-1-of-2">
+    <!-- modal header -->
+    <template v-slot:header>
```

Comment:
We had a vote about this a while ago https://demosdeutschland.slack.com/archives/C5BA8ABB2/p1628581842036700 and decided to use the long form. I still think it's the better solution because it's way easier to search for `v-slot` and in my opinion, `#` is not specific enough. But feel free to start a new vote, by now a considerable part of the FE team did not get a say in it. :slightly_smiling_face:

Reply:
True, never thought about it being harder to find when written as `#`.
Well since I did not know about this vote I accept the result (I think the result is pretty one-sided anyway). Thanks for linking the discussion